cypress/e2e/propertyEdit.feature

```diff
@@ -1,7 +1,8 @@
+@property
 @authentication
 @common
-@property-edit
 @root
+@search
```

Comment:
Let's assume changes to search MFE are made.
In such event, why would its pipeline need to trigger this spec?